charts/local-kafka/Chart.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: v2
 name: local-kafka
 description: Local Development spinup of Strimzi-managed Kafka
 type: application
-version: 0.2.1
+version: 0.3.0
 appVersion: latest
 maintainers:
   - name: diranged
     email: matt@nextdoor.com
 dependencies:
   - name: strimzi-kafka-operator
-    version: 0.26.0
+    version: 0.28.0
```

Comment:
why not version 0.27.0?

Reply:
no particular reason, mainly its the latest and i tried it without seeing things broken.